jlibra-examples/pom.xml

```diff
@@ -0,0 +1,21 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>jlibra-examples</artifactId>
+  <packaging>jar</packaging>
+  <parent>
+    <groupId>dev.jlibra</groupId>
+    <artifactId>jlibra-parent</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+
+  <dependencies>
+    <dependency>
+      <groupId>dev.jlibra</groupId>
+      <artifactId>jlibra-core</artifactId>
+      <version>${project.version}</version>
```

Comment:
I think you can drop dependency version and it will use latest, right?

Reply:
No, the version is required unless I have specified it in depencyManagement